### ISSUE TITLE
genome background never finished

### ISSUE THREAD

**Issue by olechnwin** [fmicompbio/monaLisa#69]
Hi,

I'm trying to run monaLisa with genome background but it always crashed out in RStudio and never finished.
Any suggestions on how to to run this? I'm also trying to run it on the full JASPAR2020 motifs because I am interested to see the enriched motifs over background genome not over other bins.

Thanks!

**Comment by csoneson** [comment]
Hi @olechnwin, thanks for your interest in `monaLisa`. In order to help you, we need more information, including:

- what code are you running?
- what is the error that you get (i.e., what does "crashed out" mean)?
- can you construct a small reproducible example so that we can reproduce the issue on our side?
- which version of the package are you using (what is the output of `sessionInfo`)?
- how large is your data set?

**Comment by cxtaslim** [comment]
Hi @csoneson,

Thank you for your reply. I've figured out that I can't run monaLisa with genome background in RStudio. I think the issue is actually the size. It ran to completion if I ran it on a few of JASPAR2020 motifs. It also ran to completion if I can it with otherbins as background.
I'm happy to report though, in my case, monaLisa ran to completion with genome background if I ran it on R instead of RStudio.
For reference, I tried to run monaLisa with two sets, each set is around 1000 regions.

`se2 <- calcBinnedMotifEnrR(seqs = selseqs2,
                           pwmL = pwms,
                           background = "genome",
                           genome = BSgenome.Drerio.UCSC.danRer11,
                           genome.regions = NULL, # sample from full genome
                           genome.oversample = 2,
                           BPPARAM = BiocParallel::MulticoreParam(20),
                           verbose = TRUE)
`

Since I can run this to completion in R, I'll just stick to command line R.

Thank you! I have another question but I'll create a new post. BTW, great tool!
